pyproject.toml

```
@@ -58,6 +58,8 @@ version_file = "src/bluesky_stomp/_version.py"
 
 [tool.pyright]
 strict = ["src", "tests"]
+typeCheckingMode = "standard"
+reportMissingImports = false # Ignore missing stubs in imported modules
```

Per [docs](https://diamondlightsource.github.io/python-copier-template/main/how-to/strict-mode.html)

```suggestion
[tool.pyright]
typeCheckingMode = "strict"
reportMissingImports = false # Ignore missing stubs in imported modules
```